bundle-workflow/src/read_write_s3.py

```diff
@@ -0,0 +1,56 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import boto3
+
+
+class ReadWriteFiles:
+
+    # get_S3Objects:Used for getting the files from S3
+    # bucket_name: Name of the bucket you want to get the file from
+    # role_arn: Amazon Resource Name (ARN) of the role that you want to assume
+    # role_session_name: Name for the session
+    # bucket_bundle: Object path from where you want to get the bundle
+    # bucket_manifest: Object path from where you want to get the manifest
+    # path_to_save_bundle: Path where you want to save the bundle at
+    # path_to_save_manifest: Path where you want to save the manifest at
+    def __init__(self, role_arn, role_session_name):
+        self.role_arn = role_arn
+        self.role_session_name = role_session_name
+
+    def get_S3_objects(self, bucket_name, bucket_bundle, bucket_manifest, path_to_save_bundle, path_to_save_manifest):
```

Comment:
We need to make this more generic to achieve two use cases - 
1. Download a particular S3 object
2. Download all maven dependencies from a given s3 location

Reply:
I have modified it now, as the bundle_path arg will be used for downloading the entire file/folder.